communications/GSOD-applications/2020-gsod.md

```diff
@@ -0,0 +1,57 @@
+# Google Season of Docs (GSOD) 2020
+
+* Website: https://developers.google.com/season-of-docs
+* Location: Distributed/online
+* Date: 10-11 March 2020
+* Context: Technical writers spend three months working closely with an open source community and learn about open source and new technologies. 
+* INCF call for the organisation partners: https://www.incf.org/blog/call-mentors-join-us-gsod-2020
+* Previous year’s application: https://github.com/alan-turing-institute/the-turing-way/issues/406
+
+### Title
+
+* The Turing Way project expands into a series of books on data research practices
+
+**Description:**
+
+The Turing Way is an open-source, community-led and collaboratively developed book project on making data research accessible for a wider research community ([Website](https://the-turing-way.netlify.com)). 
+We bring together individuals from diverse fields and expertise to develop practices and learning resources that can make data research comprehensible and easy to understand.
+The project is openly developed and all questions, comments, recommendations and discussion are facilitated through an [online GitHub repository](https://github.com/alan-turing-institute/the-turing-way).
+
+Our community members are researchers, engineers, data librarians, industry professionals, and experts in various domains, at all levels of seniority, from all around the world. 
+Since last year, they have co-authored chapters on [research reproducibility](https://the-turing-way.netlify.app/reproducibility/reproducibility.html) by compiling best practices, tools and recommendations used by the scientific communities worldwide.  
+
+Since the project has grown from a small team of 10 researchers to a community of 125 contributors, the scope of The Turing Way book has also expanded to include different aspects such as project design, collaboration, communication, and ethics in data research ([preview](https://deploy-preview-977--the-turing-way.netlify.app/welcome)). The project aims to provide learning and training resources on a wide array of topics and builds on case studies provided by Turing researchers, impact stories contributed by individuals and workshops delivered by the team members. 
```

Comment:
```suggestion
Since the project has grown from a small team of 10 researchers to a community of 125 contributors, the scope of The Turing Way book has also expanded to include different aspects such as project design, collaboration, communication, and ethics in data research ([preview](https://deploy-preview-977--the-turing-way.netlify.app/welcome), pull request [#977](https://github.com/alan-turing-institute/the-turing-way/pull/977)).
The project aims to provide learning and training resources on a wide array of topics and builds on case studies provided by Turing researchers, impact stories contributed by individuals, and workshops delivered by the team members. 
```